packages/vue/src/stories/releases/v0.11.0/v0.11.0.stories.mdx

```diff
@@ -0,0 +1,9 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Releases/v0.11.0 - Latest/Change log" />
+
+# v0.11.0
+
+## What is new?
+
+👉 docs for transitions in `Utilites/Transitions/Docs`
```

Comment:
Please follow our standard of writing in v.10.0 release note.

Reply:
done :)